sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/RequestValidator.java

```diff
@@ -94,6 +94,81 @@ public static void validateUpdateContent(Request contentRequestDto) {
               ERROR_CODE);
         }
       }
+      List<Map<String, Object>> assessmentData = (List<Map<String, Object>>) contentRequestDto.getRequest().get(JsonKey.ASSESSMENT_EVENTS);
+      System.out.println(assessmentData);
+      if (null != assessmentData && !assessmentData.isEmpty()) {
+        for (Map<String, Object> map : assessmentData) {
+          if (map.containsKey(JsonKey.COURSE_ID)) {
+            if (null == map.get(JsonKey.COURSE_ID)) {
```

Comment:
Use `StringUtils.isEmpty` - There is a chance of sending COURSE_ID empty string.